doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -257,6 +257,30 @@ ifdef::PAM[]
 
 == Process designer
 
+.Validation for a signal name fails if the name includes spaces or colons (`:`) [https://issues.redhat.com/browse/RHPAM-2557[RHPAM-2557]]
+
+Issue: In the process designer, the validation for a signal name fails if the name includes spaces, colons (`:`), or other special characters.
+
+Steps to reproduce:
+
+. Open a BPMN project in the process designer.
+. Add a `Signal` node on the canvas.
+. In the *Properties* panel for the node, expand *Implementation/Execution*.
+. Click the *Signal* drop-down list and select *New...*
+. In the *Signal* field, enter `Milestone 2: Order shipped`.
+. Press *Tab* on the keyboard to switch to the next field.
+
+Expected result: The field content is saved successfully.
+
+Actual result: The spaces do not appear and the field contains `Milestone2:Ordershipped`. After you press *Tab* an *Invalid characters* error message appears.
```

Comment:
Consider:
Actual Result: The *Signal* field does not contain the spaces that you entered, `Milestone 2: Order shipped` and the  *Invalid characters* error message appears.

So this is confusing because the last step is Press Tab. Actual result should be what happens after you press tab, so need to repeat it here.  If my suggestion is correct, consider using it.